source/manual/troubleshoot-development-vm.html.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Troubleshoot your development VM
+parent: "/manual.html"
+layout: manual_layout
+section: Support
+owner_slack: "#govuk-developers"
+last_reviewed_on: 2017-08-08
+review_in: 3 months
+---
+
+Here are some things that can go wrong with the development VM.
+
+## Running out of disk space
+
+You may run out of disk space when replicating data into your development
+environment.
+
```

Comment:
Ah yeah. I think I'll leave it for now, as I'm not sure exactly what to say, but feel free to add it in in a follow up.